main.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'app'
+
+def main
+  app = App.new
+  app.menu_choice
+end
+
+main
```

Comment:
When I try to load your app I immediately get an error.

![image](https://user-images.githubusercontent.com/38088818/181789771-e49cb253-0785-42a1-9b1b-1ef0bc2fe9b7.png)

- This is because you used a capitalized `Decorator` instead of small letters `decorator` to require the decorator file in `capitalize_decorator.rb`

Same thing for the `trimmer_decorator.rb`

![image](https://user-images.githubusercontent.com/38088818/181790794-50b97f34-5f2b-437a-933b-f671eb74f7ca.png)

![image](https://user-images.githubusercontent.com/38088818/181790178-226897dd-b5b6-40fd-9bef-294be3358829.png)

- Kindly fix this so that your app runs properly.